Documentation for Vermögensbegriff (#416)
iza-institute-of-labor-economics/gettsim: docs/gettsim_objects/means_testing.rst

.. _means_testing:

> Assets for means testing

Reviewer: ```suggestion
Assets considered for means testing for various transfers
```

========================

> | The table below gives an overview of all tangible and intangible assets which are deducted from the overall assets of a household when performing means testing for the respective transfer.

Reviewer: ```suggestion
| The table below gives an overview of all tangible and intangible assets which are considered when performing means for several transfers. A cross indicates that the asset class is not considered and, hence, deducted from the overall assets of a household.
```

> | This documentation shall help to understand the composition of the :ref:`basic input variable <input_variables>` 'vermögen_bedürft_hh'.

Reviewer: ```suggestion
| This documentation shall help to understand the composition of the :ref:`basic input variable <input_variables>` 'vermögen_bedürft_hh'. Despite small differences over the transfers we decided, for now, to require only one wealth variable as input and use it for all transfers.
```

.. note::
   | ALGII = Grundsicherung für Arbeitslose
   | GSA/E = Grundsicherung im Alter/ bei Erwerbsminderung
   | WoGe = Wohngeld
   | KiZu = Kinderzuschlag

>    HS = hardship case

Reviewer: HC?

Also explain a bit what this mean.

+-----------------------------------------------+---------+---------+---------+---------+
| Not counted as assets                         |  ALGII  |  GSA/E  |  WoGe   |  KiZu   |
+===============================================+=========+=========+=========+=========+
| Appropriate household goods                   |    x    |    x    |    x    |    x    |
+-----------------------------------------------+---------+---------+---------+---------+
|| adequate motor vehicle for each              ||   x    || x (HC) ||   x    ||   x    |
|| person capable of working                    ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+
|| Assets for retirement provision and          ||   x    ||   x    ||   x    ||   x    |
|| retirement assets                            ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+

> || owner-occupied house/apartment of            ||   x    ||   x    ||   x    ||   x    |

Reviewer: Make sure that the descriptions all start with a capital letter and don't have any capital letters thereafter.

|| appropriate size                             ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+
|| Assets for obtaining/maintaining a house     ||   x    ||   x    ||   x    ||   x    |
|| property for people in need of care          ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+
|| Assets to build up/secure a livelihood       ||   x    ||   x    ||   x    ||   x    |

> || or Establishment of a household              ||        ||        ||        ||        |

Reviewer: Make sure that the descriptions all start with a capital letter and don't have any capital letters thereafter.

+-----------------------------------------------+---------+---------+---------+---------+
| Family and heirloom pieces                    | x (HC)  |    x    |         | x (HC)  |
+-----------------------------------------------+---------+---------+---------+---------+
|| Property and rights, if realization would    ||   x    ||   x    ||        ||   x    |
|| be uneconomical or a particular hardship     ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+
|| Items to start/continue vocational           ||   x    ||   x    ||   x    ||   x    |
|| training/employment                          ||        ||        ||        ||        |
+-----------------------------------------------+---------+---------+---------+---------+
| Items for the satisfaction of spiritual needs ||        ||   x    ||   x    ||        |
+-----------------------------------------------+---------+---------+---------+---------+

.. seealso::

>     Here you can check out the legal bases for the definition of assets for each transfer.

Reviewer: ```suggestion
    See the following links for legal bases for the definition of assets for each transfer.
```
